### PR TITLE
App Navigation

### PR DESCRIPTION
**Pull request by eduardoccmelo**
Create static pages and basic navigation through them.

### CODE REVIEW

File: src/components/MyTrips.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+export default function MyTrips() {
+  return (
+    <div className="myTrips">
+      <h2>My Trips</h2>
+      <Link to="/myTrips/:id">Trip 1</Link>
+      <Link to="/myTrips/:id">Trip 2</Link>
+      <Link to="/myTrips/:id">Trip 3</Link>
+      <Link to="/myTrips/:id">Trip 4</Link>
+      <Link to="/myTrips/:id">Trip 5</Link>
+      <p></p>
```
Comment (FriederikeM) [inline src/components/MyTrips.js:12]: that < p > looks lonely

File: src/components/NewTrip.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { useHistory } from "react-router";
+import { Link } from "react-router-dom";
+
+export default function NewTrip() {
+  const history = useHistory();
```
Comment (FriederikeM) [inline src/components/NewTrip.js:5]: i see you introduced useHistory, but are not using it (yet) in this file

**Review by FriederikeM — APPROVED** [review]
Looks good so far, you have gotten a lot done already 💯  just left two little comments

File: src/App.css [hunk 3]
```diff
@@ -1,3 +1,12 @@
 .App {
   text-align: center;
 }
+
+.home,
```
Comment (namirsab) [inline src/App.css:5]: I see you are setting the css for each of the subcomponents of the `App` component in the `App.css` file. 
I'd move css related to each component to its own file.

File: src/components/EditTrip.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { useHistory } from "react-router";
+
+export default function EditTrip() {
+  const history = useHistory();
+  return (
+    <div className="editTrip">
```
Comment (namirsab) [inline src/components/EditTrip.js:6]: Some CSS conventions info. I link here a comment i put to Rikki.
https://github.com/FriederikeM/routinely/pull/9#discussion_r620908613

File: src/components/EditTrip.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+      <h2>Edit Page</h2>
+      <button
+        onClick={(e) => {
```
Comment (namirsab) [inline src/components/EditTrip.js:9]: You are not using the `e` parameter here. You could just remove it (probably eslint is telling you that as well if you check)

**Review by namirsab — COMMENTED** [review]
Cool Eduardo!
Just a few comments, nothing too critical

**Pull request merged**